lib/clicker_game/clicker_game_page.dart

```diff
@@ -58,76 +59,85 @@ class _ClickerGamePageState extends State<ClickerGamePage> {
-                  child: Assets.images.cloudsBackground.image(fit: BoxFit.fill),
-                ),
-                Align(
-                  alignment: Alignment.bottomCenter,
-                  child: ConstrainedBox(
-                    constraints: BoxConstraints(
-                      maxWidth: constraints.maxWidth > _maxGameWidth
-                          ? _maxGameWidth
-                          : constraints.maxWidth,
-                      maxHeight: constraints.maxHeight > _maxGameHeight
-                          ? _maxGameHeight
-                          : constraints.maxHeight,
-                    ),
-                    child: Material(
-                      elevation: 9,
-                      clipBehavior: Clip.hardEdge,
-                      child: GameWidget(
-                        game: _game,
-                        overlayBuilderMap: {
-                          GameHUD.id: (_, ClickerGame game) => GameHUD(
-                                game: game,
-                                handleRightButton: _handleBackButton,
-                              ),
-                          GameStartOverlay.id: (context, __) =>
-                              GameStartOverlay(
-                                onPressed: () => _handleGameStart(context),
-                              ),
-                          TimerReductionEffect.id: (context, __) =>
-                              TimerReductionEffect(
-                                text: '-5',
-                                onAnimationEnded: () {
-                                  context.read<TimerCubit>().penalty = 5;
-                                  _game.overlays
-                                      .remove(TimerReductionEffect.id);
-                                },
-                              ),
-                          TutorialOverlay.id: (context, __) => TutorialOverlay(
-                                onBackButtonPressed: _handleBackButton,
-                                onGameStart: () =>
-                                    _handleTutorialCompleted(context),
-                              ),
-                        },
-                        backgroundBuilder: (context) => Container(
-                          // color: FlutterGameChallengeColors.blueSky,
-                          color: const Color(0xFF72A8CD),
+          child: LayoutBuilder(
+            builder: (context, constraints) {
+              return Stack(
+                children: [
+                  Align(
+                    alignment: Alignment.topCenter,
+                    child:
+                        Assets.images.cloudsBackground.image(fit: BoxFit.fill),
+                  ),
+                  Align(
+                    alignment: Alignment.bottomCenter,
+                    child: ConstrainedBox(
+                      constraints: BoxConstraints(
+                        maxWidth: constraints.maxWidth > _maxGameWidth
```

Comment:
leaving it unconstrained let the content fit available space and since we don't have much of the restrictions on minimal size, I would prefer leave it as is